### PR TITLE
Refactor with hooks

### PR DESCRIPTION
**Pull request by daudi13**
## Description

In this project you will refactor the Calculator component you developed in the past projects. Instead of using class based components, you will use now hooks.

- Refactored the calculator component from class based component to functional component using Hooks

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### HTML/CSS requirements

- [x] Followed best practices for HTML & CSS
- [x] Followed best practices for JavaScript

### Project requirements

- [x] Refactor the Calculator component (`components/Calculator.js`) from a class based component to a functional component with hooks.
- [x] After the refactor is done, test the app in the browser. It should keep the same functionality.

---

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Math magician is a website for all fans of mathematics. It is a single Page App
 - [x] Project 1: Setup
```
Comment (Meltrust) [inline README.md:11]: The [Microverse readme template](https://github.com/microverseinc/readme-template) is a requirement for most projects at Microverse, according to the [Professional Repo Basic Rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) general requirements.

Look:

image


- [x] On your readme, please kindly implement the [Microverse readme template](https://github.com/microverseinc/readme-template).  All the sections are required except for "live demo" and "tests" if not required.

 Hint:  You can include the section that you already have written.

File: src/components/ButtonInfo.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+function Buttons() {
+  const ButtonData = [
+    [
+      { name: 'AC' },
+      { name: '+/-' },
+      { name: '%' },
+      { className: 'orange', name: '/' },
```
Comment (Meltrust) [inline src/components/ButtonInfo.js:7]: - [x] The calculator can't divide.  All divisions result in a 0.

## Explantation and hints

On `operate.js`, the wrong symbol is being used for divisions:

image

Just change it for the correct symbol `/`:

image

Now, it should work! Go ahead and correct the bug now that you know where it is located 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @daudi13,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice styling✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello @daudi13 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**